BUG: Creating unnecessary extra column in csv format

## Current Behaviour
Converting from the JSON format the CSV string currently the format is looking like 
```js
const data = `
language, rating,
javascript, 5,
java, 5, 
`
```

## Expected behavior 
The expected CSV string format should be something like this
```js
const data = `
language, rating
javascript, 5
java, 5
`
```

## Problem 
Currently, there is a comma at the end of each line which is creating an extra column. 

## Probable solution 
Removing the comma from each line should fix this issue. 
This function is responsible for generating columns in CSV format https://github.com/Ninja-Developers/jsv-convert/blob/c87e20c6b019d350eb1e3496c9479a4db10a2217/src/core.ts#L34
If we can just remove the last character after the string is generated and before concating it with the main string, we will be able to fix this issue. 

## This is the complete code responsible for generating the columns 
```ts
 generateColumns(): Array<string> {
        let columns: Array<string> = []

        this.data.forEach((el: any) => {
            let row: string = ''
            this.fields.forEach((f: string) => {
                if (el[f]) {
                    row += el[f] + ','

                } else {
                    row += ','
                }
            })
            columns.push(row)
        })

        return columns
    }

    generateCsvFormat(): string{
        let fieldString: string = ''
        this.fields.forEach((el: string) => {
            fieldString+=el+','
        })
        let colStr: string = ''
        this.columns.forEach((el: string) => {
            colStr+=el+"\n"
        })
        return `${fieldString}
${colStr}`
    }

```

@Souvikns Can you please tell me how to test the code? I am getting error on running `npm test`.

 FAIL  test/test.js
  ● Test suite failed to run

    Cannot find module '../index' from 'test.js'

    > 1 | const {prepareTable} = require('../index')
        |                        ^
      2 |
      3 | test('Fake Test',()=>{
      4 |     expect(true).toBeTruthy()

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:296:11)
      at Object.<anonymous> (test/test.js:1:24)

--- a/src/core.ts
+++ b/src/core.ts
@@ -33,15 +33,17 @@ class FileConverter {
 
     generateColumns(): Array<string> {
         let columns: Array<string> = []
+        const fieldsCount = this.fields.length;
 
         this.data.forEach((el: any) => {
             let row: string = ''
-            this.fields.forEach((f: string) => {
+            this.fields.forEach((f: string, i: number) => {
                 if (el[f]) {
-                    row += el[f] + ','
-
-                } else {
-                    row += ','
+                    row += el[f]
+                }
+                // Adding comma
+                if (i < fieldsCount - 1) {
+                  row += ','
                 }
             })
             columns.push(row)

